README.md

```diff
@@ -3,49 +3,88 @@
 This project is about harvesting from external sources to a CKAN (local or remote) instance.  
 This project uses a custom Python library ([ckan-harvesters](https://pypi.org/project/ckan-harvesters/))in order to read and validate different types of data sources.  
 
-## data.json files
+Install locally
 
-[More info](harvester_ng/datajson/README.md)
+```
+python3 -m venv /path-to-harvester-env
+source /path-to-harvester-env/bin/activate
+pip install -r requirements.txt
+```
 
-## CSW
+## Run with CKAN 
 
-[More info](harvester_ng/csw/README.md)
+First copy `.env.example` to `.env` file
+
+```
+# add "ckan" (to use ckan:5000) into "webserver" container's hosts file
+export HOST_IP=`ip -4 addr show scope global dev docker0 | grep inet | awk '{print \$2}' | cut -d / -f 1`
+docker-compose up --build
```

Comment:
we said we won't be building any docker images here.

Reply:
[here](https://github.com/datopian/ckan-ng-harvest/commit/61ed522ab0d8d6275fe614d4f01b68cabac4e6c2). I'll check more details about building